libcaf_io/caf/io/middleman.hpp

```diff
@@ -156,7 +156,7 @@ class CAF_IO_EXPORT middleman : public actor_system::module {
   template <class Handle>
   expected<Handle>
   remote_spawn(const node_id& nid, std::string name, message args,
-               duration timeout = duration(time_unit::minutes, 1)) {
+               timespan timeout = timespan{60000000000}) {
```

Comment:
Why not use chrono? That would be more readable.
```suggestion
               timespan timeout = timespan{std::chrono::minutes{1}}) {
```